libs/gl-client/src/node/service.rs

```diff
@@ -136,6 +142,11 @@ impl Service<Request<BoxBody>> for AuthService {
                 .headers
                 .insert("glts", engine.encode(ts).parse().unwrap());
 
+            // Runes already come base64 URL_SAFE encoded.
+            parts
+                .headers
+                .insert("glrune", rune.parse().expect("Could not parse rune"));
+
```

Comment:
Just passing the rune through here, so the Rune must be committing to the pubkey, and the signer needs to verify that the pubkey for the signature and the commitment in the rune match.

Reply:
Yes, validating the rune against the public key -> signature -> request should be the last part that is missing for this pull request.